### PR TITLE
Update readme for TAP beta2

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,5 +5,35 @@ This is a sample of a Java Spring app that works with Tilt and the Tanzu Applica
 ## Dependencies
 1. [kubectl CLI](https://kubernetes.io/docs/tasks/tools/)
 1. [Tilt version >= v0.22.6](https://docs.tilt.dev/install.html)
-1. [Tanzu CLI apps plugin v0.2.0](https://github.com/vmware-tanzu/tanzu-cli-apps-plugins)
-1. A cluster with Tanzu Application Platform, the Inner Loop Supply Chain, and their dependencies. Download these from Tanzunet.
+1. Tanzu CLI and the apps plugin v0.2.0 which are provided as part of [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform)
+1. A cluster with Tanzu Application Platform, and one of the provided supply chains, plus its dependencies. These supply chains are part of [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform).
```
Comment (lisamburns) [inline README.md:9]: We have only tested and can support that debug / live-reload works with the Default Supply Chain, not any supply chain.

**Review by lisamburns — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (trisberg) [reply README.md]: OK, I have updated the readme and also added Apache License 2.0

File: README.md [hunk 2]
```diff
@@ -5,5 +5,35 @@ This is a sample of a Java Spring app that works with Tilt and the Tanzu Applica
...
+    Stop! tap-beta2 might be production.
+    If you're sure you want to deploy there, add:
+        allow_k8s_contexts('tap-beta2')
+    to your Tiltfile. Otherwise, switch k8s contexts and restart Tilt.
+    ```
+
+Once the workload is up and running you can access the app on the local port that Tilt is forwarding:
+
+```
+curl -w'\n' localhost:8080
```
Comment (lisamburns) [inline README.md:32]: Sorry I didn't catch this in last review - the Tiltfile in this repo does not configure port-forwarding, so the way to access the app is through its knative service url.

Comment (lisamburns) [reply README.md]: cool glad you decoupled this to allow the rest to move forward. Feel free to open a separate issue for the curl thing. I'm not sure why it worked for you.

**Pull request merged**